tsconfig.json

```diff
@@ -22,10 +22,15 @@
       "es2019",
       "dom",
       "dom.iterable"
-    ]
+    ],
+    "paths": {
+      "posthog-js": [
+        "./src/@types/posthog.d.ts"
+      ]
+    }
```

Comment:
This shouldn't be needed if the module is named correctly e.g https://github.com/matrix-org/matrix-react-sdk/blob/4993fc3e7a2a2de5522f10ae5fdbd27432a40df5/src/@types/diff-dom.ts#L17

Reply:
I'd love to not have this - the need is for this to override the problematic type definition file at `node_modules/posthog-js/dist/module.d.ts` with our own.

Alternatively, we could fork or patch posthog-js

Reply:
Ah right, it ships its own (badly-dep'd) types 🤦